src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -39,6 +39,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val defines = PrefixSetting("-Dproperty=value", "-D", "Pass -Dproperty=value directly to the runtime system.")
   val toolcp = PathSetting("-toolcp", "Add to the runner classpath.", "")
   val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
+  val strict = BooleanSetting("-strict", "Use strict type rules, which means some formerly legal code does not typecheck anymore.")
```

Comment:
Is there a document that describes what Scala code can be used with Dotty without any changes and what requires migration?